google-cloud-speech/docs/toc.json

```diff
@@ -32,26 +32,12 @@
     {
       "title": "Speech",
       "type": "google/cloud/speech",
+      "patterns": ["\/speech", "google\/protobuf", "google\/rpc"],
       "nav": [
-        {
-          "title": "Project",
-          "type": "google/cloud/speech/project"
-        },
-        {
-          "title": "Audio",
-          "type": "google/cloud/speech/audio"
-        },
-        {
-          "title": "Operation",
-          "type": "google/cloud/speech/operation"
-        },
-        {
-          "title": "Result",
-          "type": "google/cloud/speech/result"
-        },
         {
```

Comment:
Should the changes made here also be copied to the `google-cloud` `toc.json` in this PR, to avoid an issue similar to #1969?

Reply:
Thanks for the catch. Updated.